README.md

```diff
@@ -117,6 +117,7 @@ use_global = true
 ### Working with Configurations
 
 - Words added with "Add to dictionary" are stored in the project configuration
+- Words added with "Add to global dictionary" are stored in the global configuration
```

Comment:
I'm a bit unsure.

Is it in a file in the global configuration folder?

Or it is in the configuration file itself ?

So this for Linux `~/.config/codebook/codebook.toml`

If so, maybe this


```suggestion
- Words added with "Add to global dictionary" are stored in the global configuration file
```

Reply:
The current wording seems fine to me! There is a section (`Global Configuration`) that goes into more detail about what the file locations actually are. That seems sufficent, but I'm probably too close to this to know if it's clear enough.

Reply:
I'm completely open to make the README or code look however,  as long as I can add words to the global words list.

I did think about maybe introducing a configuration that would make add_words add to global as well,  but figured I'd just keep them separate for the PR and see what folks want instead.   I think having 2 separate code actions isn't overwhelming.

![image](https://github.com/user-attachments/assets/b59fd69f-1554-40d5-807a-74e4cf3f9a28)